azure-cli.js

```diff
@@ -0,0 +1,51 @@
+const { exec, tryParseAzureOutput } = require("./helpers");
```

Comment:
Why?

Exec is an infrastructure layer part of the core functionality, not some supportive, helper one.